packages/experience/src/pages/Consent/ScopesListCard/index.tsx

```diff
@@ -15,15 +15,16 @@ import * as styles from './index.module.scss';
 
 type ScopeGroupProps = {
   groupName: string;
+  isAutoExpand?: boolean;
   scopes: Array<{
     id: string;
     name: string;
     description?: string;
   }>;
 };
 
-const ScopeGroup = ({ groupName, scopes }: ScopeGroupProps) => {
-  const [expanded, setExpanded] = useState(false);
```

Comment:
We usually need a `useEffect` hook to update this inner state, in case that `isAutoExpand` might change after the component is rendered

Reply:
this property is calculated based on the API response data. It should be a const default value and won't change.